Sources/iOS/Animation/SpringAnimation.swift

```diff
@@ -67,7 +67,7 @@ open class SpringAnimation {
   /// An Optional base view size.
   open var baseSize = CGSize(width: 48, height: 48) {
     didSet {
-      reload()
+//      reload() // as reloading hides button on iOS 14
```

Comment:
Like we discussed Farrukh, I am afraid that this will lead to other problems. Are you sure that this is the only solution and that it will not break things? 
Did you test that on a lower iOS version,  like iOS13?
I am really not convinced that the problem is at that line of code